srcpkgs/libinput-gestures/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'libinput-gestures'
+pkgname=libinput-gestures
+version=2.38
+revision=1
```

Comment:
add `noarch=yes` directly below revision=

there is no arch-specific code, the executables are a bash and python3 script